compiler/aarch64/codegen/BinaryEvaluator.cpp

```diff
@@ -615,6 +615,63 @@ OMR::ARM64::TreeEvaluator::vsubEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    return inlineVectorBinaryOp(node, cg, subOp);
    }
 
+TR::Register *
+OMR::ARM64::TreeEvaluator::vmulInt64Helper(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   /*
+    * Vector mul instruction on aarch64 does not support 64-bit interger element.
+    *
+    * Suppose we have two long integers (64 bits), AB and CD, where each of A, B, C, and D is one word (32 bits).
+    * To get their product, AB * CD, we can do the following:
+    *
+    *  AB * CD = B*D + 2^32*(A*D + B*C) + 2^64*(A*C)
+    *
+    *  Since we are only keeping the lower 64 bits of the result, this is equivalent to:
+    *
+    *  AB * CD = B*D + 2^32*(A*D + B*C)
+    *
+    *  To perform (A*D + B*C), we reverse elements in AB and do 32-bit element wise multiplication for BA and CD.
+    *
+    */
+   TR::Node *firstChild = node->getFirstChild();
+   TR::Node *secondChild = node->getSecondChild();
+   TR::Register *lhsReg = cg->evaluate(firstChild);
+   TR::Register *rhsReg = cg->evaluate(secondChild);
+
+   TR::Register *tmp1Reg = cg->allocateRegister(TR_VRF);
+   TR::Register *tmp2Reg = cg->allocateRegister(TR_VRF);
+   TR::Register *productReg = cg->allocateRegister(TR_VRF);
+
+   TR_ARM64ScratchRegisterManager *srm = cg->generateScratchRegisterManager();
```

Comment:
`srm` is used nowhere in this function.

Reply:
Thanks for pointing it out. I forgot to remove this line. Fixed.